### PR TITLE
Dao interfaces

### PR DESCRIPTION
**Pull request by dmytrokohut**

### CODE REVIEW

File: src/main/java/com/dkohut/chatservice/common/interfaces/IChatRoomDAOService.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+public interface IChatRoomDAOService {
+
+	/**
+	 * Select chat rooms in given range, sorted by created_at column.
+	 * 
+	 * @param Integer firstRow
+	 * @param Integer numberOfRows 
+	 * @return List<ChatRoom>
+	 */
+	List<ChatRoom> selectInRange(Integer firstRow, Integer numberOfRows);
```
Comment (koropatva) [inline src/main/java/com/dkohut/chatservice/common/interfaces/IChatRoomDAOService.java]: bad name of method. Not clear why we need this method at all

File: src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	/**
+	 * Select messages of one chat in given range, sorted by created_at column.
+	 * 
+	 * @param Integer firstRow
+	 * @param Integer numberOfRows
+	 * @param Integer chatRoomId
+	 * @return List<Messages> 
+	 */
+	List<Message> select(Integer firstRow, Integer numberOfRows, Integer chatRoomId);
```
Comment (koropatva) [inline src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java]: Not clear name

File: src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	 */
+	Integer create(Message message);
+	
+	/**
+	 * Delete existing message.
+	 * 
+	 * @param Integer id
+	 * @return String
+	 */
+	String delete(Integer id);
```
Comment (koropatva) [inline src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java]: Not clear why we return `String`

File: src/main/java/com/dkohut/chatservice/common/interfaces/IUserDAOService.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	 */
+	Integer create(User user);
+	
+	/**
+	 * Update existing user.
+	 * 
+	 * @param User user
+	 * @return String
+	 */
+	String update(User user);
```
Comment (koropatva) [inline src/main/java/com/dkohut/chatservice/common/interfaces/IUserDAOService.java]: The same here why `String`?

File: src/main/java/com/dkohut/chatservice/common/interfaces/IChatRoomDAOService.java (hunk 1 above)
Comment (dmytrokohut) [reply src/main/java/com/dkohut/chatservice/common/interfaces/IChatRoomDAOService.java]: Method name was changed.

File: src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java (hunk 2 above)
Comment (dmytrokohut) [reply src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java]: Method name was changed.

File: src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java (hunk 3 above)
Comment (dmytrokohut) [reply src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java]: It was a bad idea. The return type was changed.

File: src/main/java/com/dkohut/chatservice/common/interfaces/IUserDAOService.java (hunk 4 above)
Comment (dmytrokohut) [reply src/main/java/com/dkohut/chatservice/common/interfaces/IUserDAOService.java]: It was a bad idea. The return type was changed.

**Pull request closed without merging**